### DISCUSSION TITLE
Idea

### DISCUSSION THREAD

**Discussion by FxckingAngel** [ItzSudhan/discordjs-activity#31] [Ideas]
I want to make a discord app like the youtube one you made but different. Like when I use put a link or text for the music. It auto steam it.
My discord:🖤Witch Of Greed🖤#2190

**Comment by Jotarokujo10525** [comment]
Explain further the `Like when I use put a link or text for the music. It auto steam it.`
I don't understand the "auto steam it"
Also if you're talking about `YouTube Together`, it's a experimental project of **DISCORD** itself, so it has its limitations and its not fully out yet. 

Also, clarification. The `I want to make a discord app` means *you* want to code it?
